### DISCUSSION TITLE
System data

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#784] [Q&A]
Hello, DP developers

Now, we use some data to train, and I don't know how many tracks it takes.An OUTCAR can contain all the information with different structures?It is how much data a system needs to fit. Besides looking at loss, is there any standard or system data

I am looking forward to your reply

**Comment by Ericwang6** [comment]
The program will randomly select frames from your training system, so that theoretically if your training steps are longer enough, all frames in the training set can be fed into the training process. You can use dpdata to check the number of frames in your training data.
And I'm afraid I do not understand your second question, do you mean that you want to know some methods to evaluate the performance of your model?
